CHANGES.md

```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+# 1.4.14 / 0.20.14 (libraries)
+
+### New
+
+- Added a new tooltip to asset runs to either view the asset list or lineage
+
+### Bugfixes
+
+- [ui] Fixed an issue where Re-execution a job wouldn’t navigate to the newly created run
```

Comment:
Fixed an issue where re-executing a job from the jobs page wouldn’t navigate to the newly created run

Reply:
I'm bad at wording 😅